155zc.c

```diff
@@ -0,0 +1,62 @@
+typedef struct
+{
+    int capacity;
+    int topofstack;
+    int *array;
+} MinStack;
+
+/** initialize your data structure here. */
+#define EMPTYTOS (-1)
+MinStack* minStackCreate(int maxSize)
+{
+    MinStack *S;
+    S = malloc(sizeof(MinStack)*maxSize);
+    if(S == NULL)
```

Comment:
Short conditional statements may be written on one line if this enhances readability. You may use this only when the line is brief and the statement does not use the else clause.

https://google.github.io/styleguide/cppguide.html#Conditionals